src/sql/azdata.d.ts

```diff
@@ -3433,6 +3433,7 @@ declare module 'azdata' {
 		isReadOnly: boolean;
 		width: number | string;
 		categoryValues?: CategoryValue[];
+		requiredIndicator?: boolean;
```

Comment:
New changes to azdata.d.ts should always be added to azdata.proposed.d.ts first. The definitions will be merged during build time so you can just add it like this in the proposed file : 

```
export interface DeclarativeTableColumn {
		requiredIndicator?: boolean; 
}
```